x/ccv/provider/types/legacy_proposal.go

```diff
@@ -239,6 +240,55 @@ func (sccp *ConsumerRemovalProposal) ValidateBasic() error {
 	return nil
 }
 
+// NewConsumerModificationProposal creates a new consumer modification proposal.
+func NewConsumerModificationProposal(title, description, chainID string,
```

Comment:
Does this need a `[Deprecated]` comment?

Reply:
@bermuell ?

Reply:
I would add a `[Deprecated]` comment. It should be deprecated after we get Permissionless.